app/assets/src/components/views/bulk_download/ReviewStep.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { get, find } from "lodash/fp";
+import PrimaryButton from "~/components/ui/controls/buttons/PrimaryButton";
+
+import cs from "./review_step.scss";
+
+class ReviewStep extends React.Component {
+  renderDownloadField = (field, value) => {
+    const { downloadTypeMetadata } = this.props;
+    const fieldDisplayName =
+      get("display_name", find(["type", field], downloadTypeMetadata.fields)) ||
+      "";
+
+    return (
+      <div className={cs.field}>
+        <div className={cs.name}>{fieldDisplayName}</div>
+        <div className={cs.value}>{value}</div>
+      </div>
+    );
+  };
+
+  render() {
+    const { selectedDownload, downloadTypeMetadata, onBackClick } = this.props;
+
+    return (
+      <div className={cs.reviewStep}>
```

Comment:
I am curious if there aren't any small element that are reusable on this component? 
This seems like yet another modal that will have slightly different styles.
Do you see any opportunities to consolidate?

Reply:
There are very small opportunities. I could factor out the <Footer>. But the overlap is quite small overall. It doesn't seem worth it to me right now, if the overlap became more substantial I would consolidate then.

Reply:
This was more of a philosophical/long term strategy question. Maybe part of a larger consolidation effort with @happyimadesignr.